README.md

```diff
@@ -1569,6 +1569,8 @@ Cassandra takes a snapshot your tables and ships them to a remote location. Once
 
 You can take a complete snapshot of your DC/OS Cassandra ring and upload the artifacts to S3 or to Azure.
 
+**Note:** These instructions describe how to back up the _data_ in your Cassandra ring. You must back up your Cassandra _schemas_ manually.
```

Comment:
Schema evolution and migration in Cassandra (or any NoSQL or RDBMS for that matter) is a complicated subject. Until we arrive on a decision on how opinionated we want to be on the matter, and on what capabilities we want to provide, I suggest that the above is sufficient. We should let migration and evolution occur out of band and not enforce anything until we have a plan that can be proven to "first do no harm".